index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+
+</head>
+<body>
+    <div id="top_bar"> 
```

Comment:
it is considered a good practice to use HTML5 semantic tags instead of div tags whenever it is possible, for example in line 12 it is better to replace div (with id="top_bar")  with header element.